### DISCUSSION TITLE
请求添加ddns-scripts-huaweicloud至官方仓库

### DISCUSSION THREAD

**Discussion by sxlehua** [immortalwrt/immortalwrt#1506] [Ideas]
目前在使用immortalwrt的ddns，发现没有华为云的ipk，于是自己写了更新脚本。
当前脚本：https://github.com/sxlehua/ddns-scripts-huaweicloud
请问如何添加到官方仓库，方便大家使用？

**Comment by 1715173329** [comment]
1. Rewrite all commens into English.
2. Add it to ddns-scripts package and submit to upstream OpenWrt.

**Reply by sxlehua** [reply]
OpenWrt上游代码修改好了，方便的话给处理一下。
https://github.com/openwrt/packages/pull/25120
